source/_patterns/02-organisms/04-loancards/12-loancards-2-array.mustache

```diff
@@ -0,0 +1,20 @@
+<div class="list-view-header">
```

Comment:
I think we can re-use the existing file here and just use a switch similar to what I setup for the list view buttons.